tools/upload-sketch.rshell

```diff
@@ -4,6 +4,8 @@
 cp boot.py /flash
 cp main.py /flash
 cp settings.py /flash
+cp hiveeyes/webserver.py /flash/lib/hiveeyes/webserver.py
+cp terkin/datalogger.py /flash/lib/terkin/datalogger.py
```

Comment:
I see that this is convenient for the current scope you are working on. However, this would increase turnaround time for all others running `make sketch-and-run` with the intention of just uploading the main sketch incl. `settings.py`.

However, I totally see your point to add more convenience to that. While `make install-framework` and `make install-recycle` both would sync the `hiveeyes`-folder, they will also attempt to sync a lot of other stuff not having been touched at all.

Hmmm, there should be something like `make sketch-and-run`++ or `make recycle`-only-changed in the long run.

So, on the one hand please keep yourself happy by amending this very file. On the other hand, we are humbly asking not to commit any changes to it when intending to upstream these changes by creating pull requests.